public/index.html

```diff
@@ -243,6 +243,28 @@ <h1>Do you want early access to CoinFi News?</h1>
       }
     }
     tryHook();
+
+    $(document).ready(function() {
+				$('video.product-video').each(function(index) {
```

Comment:
@anthotsang Sorry for forgetting to mention during onboarding - the team has agreed to use spaces instead of tabs for indents.